### ISSUE TITLE
No Authorization header on getMe()

### ISSUE THREAD

**Issue by J4ck4l0pe** [paulvanbladel/aurelia-auth#93]
I am trying to get my user data through the getMe() function, but it doesn't send the header.

config:
`var configForDevelopment = {
  httpInterceptor: true,
  loginOnSignup: false,
  baseUrl: '/',
  loginRedirect: '/',
  logoutRedirect: '/',
  signupRedirect: false,
  loginUrl: '/auth/login',
  signupUrl: '/auth/register',
  profileUrl: '/api/v2/users/me',
  loginRoute: '#/login',
  signupRoute: '#/register',
  tokenRoot: false,
  tokenName: 'token',
  tokenPrefix: 'aurelia',
  responseTokenProp: 'access_token',
  authHeader: 'Authorization',
  authToken: 'bearer',
  withCredentials: true,
  platform: 'browser',
  storage: 'localStorage'
};`

<img width="365" alt="headers" src="https://cloud.githubusercontent.com/assets/12096703/12865865/ba23c6ac-ccb9-11e5-8de2-a071f88de0b9.png">

**Comment by paulvanbladel** [comment]
and is the token sent for other requests you make?

**Comment by michalru3ch** [comment]
Hi guys.

I have similar problem and it seems to be caused by typo in authenticate.js. Please check line 77 - function getPayload tries to return ```parsed``` but you probably mean ```_parsed``` as defined on line 72.

Current implementation:
```
{
	key: 'getPayload',
	value: function getPayload() {

		var token = this.storage.get(this.tokenName);

		if (token && token.split('.').length === 3) {
			var base64Url = token.split('.')[1];
			var base64 = base64Url.replace(/-/g, '+').replace(/_/g, '/');

			try {
				var _parsed = JSON.parse(decodeURIComponent(escape(window.atob(base64))));
			} catch (error) {
				return;
			}

			return parsed;
		}
	}
}
```

@J4ck4l0pe for now just replace it locally and we'll wait for update

**Comment by RWOverdijk** [comment]
@michalru3ch True. We've [fixed that one](https://github.com/SpoonX/aurelia-auth/blob/master/src/authentication.js#L41,L54) already. A similar fix shouldn't be hard to write for this repository. Want to PR?

**Comment by michalru3ch** [comment]
You can do it; be my guest. I haven't cloned this library (just using it in my Aurelia project).

**Comment by paulvanbladel** [comment]
No problem, I'll fix it.
Thanks for reporting the bug.
paul.

**Comment by paulvanbladel** [comment]
fixed in 0.12.2

**Comment by J4ck4l0pe** [comment]
Thanks for fixing. I also called the `getMe()` before the `fetchConfig.configure()`. This might also have contributed to the problem.
